rdwatch/scoring/models/taskexport.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from rdwatch.core.models.annotation_exports import AbstractFileExport
+
+
+class AnimationSiteExport(AbstractFileExport):
+    site_evaluation = models.CharField(max_length=255)
+
+    arguments = models.JSONField(null=True, default=None)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+
+class AnimationModelRunExport(AbstractFileExport):
+    configuration = models.CharField(max_length=255)
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    arguments = models.JSONField(null=True, default=None)
```

Comment:
To maintain the mirrored file hierarchy between the `core` and `scoring` apps, could this module be named `annotation_exports.py`?

Reply:
I think I'll rename the core one task_exports.py.  It's really more than just annotation_exports now.  It has the base class for exporting annotations and animations.